src/test/java/org/takes/http/BkBasicTest.java

```diff
@@ -50,15 +61,20 @@
  *
  * @author Dmitry Zaytsev (dmitry.zaytsev@gmail.com)
  * @version $Id$
- * @since 0.15.2
  * @checkstyle ClassDataAbstractionCouplingCheck (500 lines)
  * @checkstyle MultipleStringLiteralsCheck (500 lines)
  * @todo #306:30min At the moment we don't support HTTP
  *  persistent connections. Would be great to implement
  *  this feature. BkBasic.accept should handle more
  *  than one HTTP request in one connection.
+ * @since 0.15.2
```

Comment:
@alexey-krylov this change not related to the ticket that's why I asked about it
about tag orders you can read here [Order of Tags](http://www.oracle.com/technetwork/articles/java/index-137868.html)